sample_cgfs/rv32i_priv.cgf

```diff
@@ -1,21 +1,413 @@
 ecall:
   config: 
     - check ISA:=regex(.*I.*); def rvtest_mtrap_routine=True 
-  opcode: 
+  mnemonics: 
     ecall: 0
 
 ebreak:
   config: 
     - check ISA:=regex(.*I.*); def rvtest_mtrap_routine=True 
-  opcode: 
+  mnemonics: 
     ebreak: 0
+    
+ 
+misa_check_combination:
+  config: 
+    - check ISA:=regex(.*I.*Zicsr.*F.*D); 
+  csr_comb:
+    (misa & 0x08) | (misa & 0x20) == 0 : 0			# setting “F”=0 and “D”=1 should results in both “F” and “D” being cleared.
+
+
+misa_mxl_check:
+  config: 
+    - check ISA:=regex(.*I.*Zicsr.*); 
```

Comment:
regex needs to have RV32 at the begining

Reply:
This cover point is written in generic terms with respect to XLEN so i think there is no need to add RV32 in regex

Reply:
I have modified by writing 32, 64 and 128 in separate labels